src/upload.py

```diff
@@ -119,12 +119,21 @@ def create_layer(
     wd: pathlib.Path,
     titiler_config: dict,
     template_env: Environment,
+    lyr_metadata: dict,
+    project_metdata: dict,
 ) -> None:
     category = layer["categoryEcosystemAccounting"].replace(" ", "_").replace(".", "")
     category_slug = f"{project}_{category}"
 
-    template = template_env.get_template("layer_description.html")
+    # Metadata dictionaries
+    data_description = lyr_metadata["dataDescription"]
+    citation = lyr_metadata["citation"]
+    temporal_scope = lyr_metadata["temporalScope"]
+    geographic_scope = lyr_metadata["geographicScope"]
+    taxonomic_scope = lyr_metadata["taxonomicScope"]
+    methodology = lyr_metadata["methodology"]
```

Comment:
There should be no need to extract the dictionaries to variables, you can send `lyr_metadata` to the template and access the dictionaries in it

Reply:
if you prefer to keep the variables for readability, you can declare such variables inside the template using:
 `{% set data_description = lyr_metadata["dataDescription"] %}` or directly on the template.render:
`data_description=lyr_metadata["dataDescription"]`

Reply:
Nice! I now declared the variables in the template.